mongoose/messages/MessageSchema.ts

```diff
@@ -6,7 +6,7 @@ import Message from "../../models/messages/Message"
 
 const MessageSchema = new mongoose.Schema<Message>({
```

Comment:
See naming convention comment from this PR: "https://github.com/Romangino/software-engineering-project-node/pull/15"